tests/drivers/bbram/generic/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright 2023 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(bbram)
+
+target_sources(app
+    PRIVATE
+        include/fixture.h
```

Comment:
Do you need to list this here? It's not a compilation unit.

Reply:
It's a neat trick I learned recently, it makes incremental builds better.

Reply:
Ah, I stand corrected, it wasn't the incremental builds, I misunderstood when it was explained to me (odd since I was noticing a difference in my build environment). Anyway, it allows IDEs to associate the headers with the libraries which gives you better suggestions for the build targets (I use CLion, but my understanding is that this also affects users of Visual Studio).